aws/resource_aws_emr_managed_scaling_policy.go

```diff
@@ -0,0 +1,147 @@
+package aws
+
+import (
+	"log"
+	// "time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/emr"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAwsEMRManagedScalingPolicy() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsEMRManagedScalingPolicyCreate,
+		Read:   resourceAwsEMRManagedScalingPolicyRead,
+		Delete: resourceAwsEMRManagedScalingPolicyDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"cluster_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"compute_limits": {
+				Type:     schema.TypeSet,
+				Required: true,
+				ForceNew: true,
+				Elem:     computeLimitsSchema(),
+			},
+		},
+	}
+}
+func computeLimitsSchema() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"unit_type": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ForceNew:     true,
+				ValidateFunc: validateAwsEmrComputeLimitsUnitType(),
+			},
+			"minimum_capacity_units": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				ForceNew: true,
+				Default:  1,
+			},
+			"maximum_capacity_units": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				ForceNew: true,
+				Default:  1,
```

Comment:
This parameter is listed as required and does not list the default of 1 in the [API Reference](https://docs.aws.amazon.com/emr/latest/APIReference/API_ComputeLimits.html)

```suggestion
				Required: true,
				ForceNew: true,
```